tools/getLBT/parameters.json.example

```diff
@@ -0,0 +1,13 @@
+{
+    "agglayerBridgeAddress": "0x..",
+    "options": {
+        "blockRange": 1000,
+        "printEvents": true,
+        "getEventsFromFile": false,
+        "concurrencyLimit": 100,
+        "printTokens": true,
+        "outputPathTokensArray": "upgrade/upgradeEtrogSovereign/tokens.json",
+        "outputPathLBT": "upgrade/upgradeEtrogSovereign/LBT.json",
+        "blockNumber": "latest",
```

Comment:
maybe by default use finalized or safe? i mean, if there is a reorg we would be starting with a wrong LBT without even knowing. As we discussed before due to the timelock wait new wrapped tokens can appear, but there is not much we can do re that.